plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -225,6 +225,9 @@
     @Parameter(property = "fabric8.namespace")
     private String namespace;
 
+    @Parameter(property = "fabric8.skipHealthChecks", defaultValue = "false")
+    private Boolean skipHealthChecks;
```

Comment:
fabric8.disableProbes might be a better name since Its Health probe and liveness probe.

if not, atleast remove s , just fabric8.skipHealthCheck :)